src/main/java/com/ifactory/press/db/solr/HitCount.java

```diff
@@ -40,14 +48,29 @@
     public ValueSource parse(FunctionQParser fp) throws SyntaxError {
         // hitcount() takes no arguments.  If we wanted to pass a query
         // we could call fp.parseNestedQuery()
+        IndexReader emptyReader = null;
+        try {
+            emptyReader = new MultiReader();
+        } catch (IOException ex) {
+            Logger.getLogger(HitCount.class.getName()).log(Level.SEVERE, null, ex);
+        }
+        Set<Term> termSet = new HashSet<Term>();

```

Comment:
Any reason to create `termSet` rather than using the `terms` var defined below on line 69? I assume this is meant to replace `terms`.

If so, then we should probably remove the instantiation below and change the `for` loop on line 78 to iterate over the correct variable.